test/template/repository.json

```diff
@@ -0,0 +1,21 @@
+{
+  "Outputs": {
+    "RepositoryBucketName": {
+      "Export": {
+        "Name": "ApkRepositoryBucketName"
+      },
+      "Value": {
+        "Ref": "Repository"
+      }
+    }
+  },
+  "Resources": {
+    "Repository": {
+      "Properties": {
+        "AccessControl": "PublicRead",
+        "BucketName": "mbj-apk"
+      },
+      "Type": "AWS::S3::Bucket"
+    }
+  }
+}
```

Comment:
There is a missing newline at the end of this file.

Reply:
That file is generated. I'll not block on this until I fix the generator centrally.